components/todo-list.js

```diff
@@ -1,19 +1,22 @@
-import React from "react";
-import styled from "styled-components";
+import React from 'react';
+import styled from 'styled-components';
 
-import TodoItem from "./todo-item";
+import TodoItem from './todo-item';
 
-export default function TodoList() {
-  return (
-    <Wrapper>
-      Todo list가 들어가는 자리입니다!
-      <TodoItem />
-    </Wrapper>
-  );
+export default function TodoList(props) {
+	const { data, onRemove } = props;
```

Comment:
```suggestion
	const { todos, onDelete } = props;
```
왜 이렇게 바뀌었는지는 ```pages/index.js```의 리뷰를 참고해주세용